frontend/src/components/module-editor/SideBarModuleOverview.tsx

```diff
@@ -81,6 +105,20 @@ const SideBarModuleOverview = (): React.ReactElement => {
       >
         New Lesson
       </Button>
+      <Modal
+        header="Edit lesson title"
+        isOpen={isOpen}
+        onConfirm={() => createLesson(title)}
+        onCancel={onClose}
+      >
+        <TextInput
+          placeholder="New Lesson"
+          onChange={setTitle}
```

Comment:
<img width="431" alt="image" src="https://user-images.githubusercontent.com/53355975/170810815-b3d1e8b4-d027-48d2-baa5-71eedbfd0c62.png">
I think it would be nice for the error to clear as soon as we input something. I think to do that, you could do something like this.

```suggestion
         onChange={(currTitle) => {
         setTitle(currTitle)
         setIsInvalid(false)
         }
```